webapp/components/ContributionForm/ContributionForm.vue

```diff
@@ -152,6 +171,12 @@ export default {
     contentLength() {
       return this.$filters.removeHtml(this.formData.content).length
     },
+    groupId() {
+      return this.group && this.group.id
+    },
```

Comment:
```suggestion
    groupId() {
      return this.group && this.group.id
    },
    showGroupHint() {
      return this.grouupId && ['closed', 'hidden'].includes(this.group.groupType)
    },
```

Reply:
Big thanks for your important hint! @Mogge